src/main/java/org/kainos/ea/dao/DatabaseConnector.java

```diff
@@ -0,0 +1,46 @@
+package org.kainos.ea.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.logging.Logger;
+
+public final class DatabaseConnector {
+
+    private static Connection conn;
+    private static final Logger LOGGER = Logger.getLogger(
+            DatabaseConnector.class.getName());
+
+    private DatabaseConnector() { }
```

Comment:
reopening

Reply:
Will log a message if the database connected successfully, and will log a message with error if connection isn't successful, I will remove it if not neccessary